docs/tutorials/intro/running_tasks_errors.rst

```diff
@@ -0,0 +1,390 @@
+    True
+    >>> error.failed_hosts['leaf01.cmh'][0].result
+    'yikes'
+    >>> error.failed_hosts['leaf01.cmh'][0].exception
+    >>>
+
+Or you can just grab the :obj:`brigade.core.task.AggregatedResult` inside the exception and do something useful with it::
+
+    >>> error.result.items()
+    dict_items([('host1.cmh', [<brigade.core.task.Result object at 0x109043518>]), ('host2.cmh', [<brigade.core.task.Result object at 0x109048c50>]), ('spine00.cmh', [<brigade.core.task.Result object at 0x1090486a0>]), ('spine01.cmh', [<brigade.core.task.Result object at 0x1090483c8>]), ('leaf00.cmh', [<brigade.core.task.Result object at 0x109048940>]), ('leaf01.cmh', [<brigade.core.task.Result object at 0x1090439e8>])])
+
+Not failing by default
+----------------------
+
+Now, let's repeat the previous example but setting ``raise_on_error=False``::
+
+    >>> from brigade.core.task import Result
+    >>> from brigade.easy import easy_brigade
+    >>> from brigade.plugins.tasks.text import print_result
+    >>>
+    >>> brigade = easy_brigade(
+    ...         host_file="hosts.yaml", group_file="groups.yaml",
+    ...         dry_run=True,
+    ...         raise_on_error=False,
+    ... )
+    >>>
+    >>>
+    >>> def task_that_sometimes_fails(task):
+    ...     if task.host.name == "leaf00.cmh":
+    ...         raise Exception("an uncontrolled exception happened")
+    ...     elif task.host.name == "leaf01.cmh":
+    ...         return Result(host=task.host, result="yikes", failed=True)
+    ...     else:
+    ...         return Result(host=task.host, result="swoosh")
+    ...
+    >>>
+    >>> b = brigade.filter(site="cmh")
+    >>>
+    >>> r = b.run(task_that_sometimes_fails)
+    >>>
+
+If ``raise_on_error=False`` the result of the task will contain a :obj:`brigade.core.task.AggregatedResult` object describing what happened::
+
+    >>> r["leaf00.cmh"].failed
+    True
+    >>> r["leaf00.cmh"].result
+    'Traceback (most recent call last):\n  File "/Users/dbarroso/workspace/brigade/brigade/core/__init__.py", line 201, in run_task\n    r = task._start(host=host, brigade=brigade, dry_run=dry_run)\n  File "/Users/dbarroso/workspace/brigade/brigade/core/task.py", line 41, in _start\n    r = self.task(self, **self.params) or Result(host)\n  File "<stdin>", line 3, in task_that_sometimes_fails\nException: an uncontrolled exception happened\n'
+    >>> r["leaf00.cmh"].exception
+    Exception('an uncontrolled exception happened',)
+    >>> r["leaf01.cmh"].failed
+    True
+    >>> r["leaf01.cmh"].result
+    'yikes'
+    >>> r["leaf01.cmh"].exception
+    >>> r["host1.cmh"].failed
+    False
+    >>> r["host1.cmh"].result
+    'swoosh'
+
+Skipping Hosts
+--------------
+
+If you set ``raise_on_error=False`` and a task fails ``brigade`` will keep track of the failing hosts and will skip the host in following tasks::
+
+	>>> r = b.run(task_that_sometimes_fails)
+	>>> r.failed
+	True
+	>>> r.failed
+	False
```

Comment:
This doesn't look correct?

Reply:
Yes, I saw that also, is there a missing call to `b.run`?

```
215     >>> r = b.run(task_that_sometimes_fails)
216     >>> r.failed
217     True
218     >>> r.failed
219     False
```

Reply:
maybe... I will correct as I move it to a jupyter notebook.